spring-integration-hazelcast/.gitignore

```diff
@@ -0,0 +1,4 @@
+/target/
+/.settings/
+/.classpath
+/.project
```

Comment:
Hi @erenavsarogullari !

Yeah... You didn't understand me.
I said add general comment "Pushed" for whole commit not for each my comment :smile: 

You should comment on the lines if you don't agree with me or have more thoughts for the individual concern